stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -807,3 +807,17 @@ extension CGFloat : _CVarArgPassedAsDouble, _CVarArgAligned {
     return native._cVarArgAlignment
   }
 }
+
+extension CGFloat : Codable {
+  @_transparent
+  public init(from decoder: Decoder) throws {
+    let container = try decoder.singleValueContainer()
+    self.native = try container.decode(NativeType.self)
```

Comment:
Doesn't this mean that archives with `CGFloat` in them are nonportable? Is that acceptable?